rcl/test/memory_tools/memory_tools.hpp

```diff
@@ -16,8 +16,8 @@
 //   https://dxr.mozilla.org/mozilla-central/rev/
 //    cc9c6cd756cb744596ba039dcc5ad3065a7cc3ea/memory/build/replace_malloc.c
 
-#ifndef MEMORY_TOOLS_HPP_
-#define MEMORY_TOOLS_HPP_
+#ifndef RCL__TEST__MEMORY_TOOLS__MEMORY_TOOLS_HPP_
+#define RCL__TEST__MEMORY_TOOLS__MEMORY_TOOLS_HPP_
```

Comment:
This is usually based on the namespace. Applying the full path here seems to be excessive.